api/src/main/resources/workbench-api.yaml

```diff
@@ -6329,6 +6329,10 @@ definitions:
         "$ref": "#/definitions/OrganizationType"
         description: 'The Organization Type of this institution if it is one of the
           enumerated OrganizationTypes, or OrganizationType.OTHER if it is not'
+      registeredTierMembershipRequirement:
```

Comment:
Can you make a note above that duaTypeEnum is now deprecated? FYI it also appears to still be in use here: https://github.com/all-of-us/workbench/blob/master/ui/src/app/pages/admin/admin-institution.tsx#L152

Reply:
Added a note. The reference you pointed will be removed in my following PR: https://github.com/all-of-us/workbench/pull/5327